src/service/config-store.ts

```diff
@@ -0,0 +1,56 @@
+import { HelpConfig } from '../help-config';
+import { ADMIN_CONFIG as adminConfig } from './configs/admin';
+import { AWARD_MANAGEMENT_CONFIG } from './configs/award';
+import { BB_HELP_CONFIG as bbHelpConfig } from './configs/bbhelp';
+import { config as bulkimportConfig } from './configs/bulkimport';
+import { config as churchConfig } from './configs/church';
+import { config as eduConfig } from './configs/edu';
+import { config as faithConfig } from './configs/faith';
+import { config as feConfig } from './configs/fe';
+import { config as fndliteConfig } from './configs/fndlite';
+import { config as gmkConfig } from './configs/grantmaking';
+import { config as marketplaceConfig } from './configs/marketplace';
+import { config as rexConfig } from './configs/rex';
+import { config as smallGroupsConfig } from './configs/smallgroups';
+import { STEWARDSHIP_MANAGEMENT_CONFIG } from './configs/stewardship';
+import { config as tcsConfig } from './configs/tcs';
+
+export type VALID_CONFIG_ID = 'admin'
+  | 'bbcrm'
+  | 'bbhelp'
+  | 'award-management'
+  | 'stewardship-management'
+  | 'chrch'
+  | 'edu'
+  | 'faith'
+  | 'fenxt'
+  | 'fenxt(fenxt.cdev)'
+  | 'fenxt(fenxt.ate)'
+  | 'fndlite'
+  | 'marketplace'
+  | 'renxt'
+  | 'smallgroups'
+  | 'tcs'
+  | 'gmk';
+
+export const CONFIG_STORE: { [k in VALID_CONFIG_ID]: HelpConfig } = {
+  admin: adminConfig,
+  'award-management': AWARD_MANAGEMENT_CONFIG,
+  bbcrm: bulkimportConfig,
+  bbhelp: bbHelpConfig,
+  chrch: churchConfig,
+  edu: eduConfig,
+  faith: faithConfig,
+  fenxt: feConfig,
+  // FE has various svcid's based on their environments for testing.
+  ['fenxt(fenxt.cdev)']: feConfig,
+  ['fenxt(fenxt.ate)']: feConfig,
+  // This is necessary as bbcrm is using the old help format, and bulkimport is a modern SPA.
+  fndlite: fndliteConfig,
```

Comment:
The comment makes me assume this has something to do with BBCRM, but this pulls up the TCS help file. Is that the correct behavior?

Reply:
this comment came from the bb-help SPA. looking through that project's history, it looks like a [commit reordered the values](https://dev.azure.com/blackbaud/Products/_git/skyux-spa-bb-help/commit/062f3a7c41275343e94c3fad3fc4da84b6f77c09?refName=refs%2Fheads%2Fmaster&path=%2Fsrc%2Fapp%2Fcomponents%2Fshared%2Fconfig-store.service.ts&_a=contents) but did not move the comment along with it. it seems like that comment is supposed to be on the `bbcrm` property based on an [earlier commit](https://dev.azure.com/blackbaud/Products/_git/skyux-spa-bb-help?_a=blame&path=%2Fsrc%2Fapp%2Fcomponents%2Fshared%2Fconfig-store.service.ts&version=GCc9a0f480111a239a9a6090cd7137418230fcf425). which makes more sense. i'll move the comment to avoid future confusion.